src/views/credentials/addCredentialModal.tsx

```diff
@@ -270,25 +342,47 @@ const CredentialFormFields: React.FC<CredentialFormFieldsProps> = ({
             name="become_password"
             onChange={event => handleInputChange('become_password', (event.target as HTMLInputElement).value)}
             ouiaId="become_password"
+            validated={errors?.become_password ? 'error' : 'default'}
           />
+          {ErrorFragment(errors?.become_password)}
         </FormGroup>
       </React.Fragment>
     )}
   </React.Fragment>
 );
+
+const ErrorFragment = (errorMessage: string | undefined) => {
+  if (errorMessage) {
+    return (
+      <FormHelperText>
+        <HelperText>
+          <HelperTextItem
+            variant={errorMessage ? 'error' : 'default'}
+            {...(errorMessage && { icon: <ExclamationCircleIcon /> })}
+          >
+            {errorMessage}
+          </HelperTextItem>
+        </HelperText>
+      </FormHelperText>
+    );
+  } else {
+    return null;
+  }
+};
```

Comment:
should this sit under components submodule (I can see other forms using this as well)?

also, what am I missing to be able to use this as `<ErrorFragment props>` instead of `{ErrorFragment(props)}`?